index.js

```diff
@@ -61,14 +59,29 @@ const GATrack = {
   },
 
   send() {
-    if (typeof window.ga === 'undefined') { // eslint-disable-line no-underscore-dangle
+    // eslint-disable-next-line prefer-rest-params
+    const args = Array.prototype.slice.call(arguments);
+
+    if (GATrack.isGA) {
+      args.unshift('send');
+    }
+
+    GATrack.sendData.apply(null, args);
+  },
+
+  sendData() {
+    if (!GATrack.isEnabled) {
       return GATrack;
     }
 
     // eslint-disable-next-line prefer-rest-params
     const args = Array.prototype.slice.call(arguments);
-    args.unshift('send');
-    ga.apply(null, args);
+
+    if (GATrack.isGA) {
+      ga.apply(null, args);
+    } else if (GATrack.isGTag) {
```

Comment:
what about GAQ? is that deprecated?

Reply:
Yep, we're only keeping it for old sites. Analytics offers the new GA by default.